cranelift/codegen/src/machinst/mod.rs

```diff
@@ -567,3 +612,37 @@ pub trait TextSectionBuilder {
     /// the bytes of the text section.
     fn finish(&mut self, ctrl_plane: &mut ControlPlane) -> Vec<u8>;
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_call_type_is_call() {
```

Comment:
No need for little tests like this on helper methods that are just `match`es on an enum -- they don't add any value but rather they are just repeating the definition.